pkgs/applications/virtualization/xen/0004-fix-oxenstored-systemd-notification-race.patch

```diff
@@ -0,0 +1,15 @@
+diff -ru xen/tools/hotplug/Linux/launch-xenstore.in xen-delay1/tools/hotplug/Linux/launch-xenstore.in
+--- xen/tools/hotplug/Linux/launch-xenstore.in	2017-05-27 10:19:32.000000000 +0200
++++ xen-delay1/tools/hotplug/Linux/launch-xenstore.in	2017-05-29 23:33:53.000000000 +0200
+@@ -66,6 +66,11 @@
+ 
+ 	systemd-notify --booted 2>/dev/null || timeout_xenstore $XENSTORED || exit 1
+ 
++	# We wait for 10 seconds here as otherwise the sd_notify() notification
++	# from the oxenstored process might not be properly registered by systemd
++	# if this process quits before the notification is sent.
++	sleep 10
```

Comment:
Actually, this code has since been dropped from the PR–see comment below, which I reproduce here:

> Interestingly, as I just found out, the patch is not needed anymore as I cannot reproduce the problematic behaviour with systemd 233. It seems that there was a problem with systemd 232, which led to the notification being dropped. Given that, it seems that the scripts in xen are correct, and the patch can be dropped–I'll drop it from the commit.